projects/m2/006-perfect-numbers/js/index.js

```diff
@@ -0,0 +1,47 @@
+const prompt = require("prompt-sync")();
+let valueChoice = 0;
+
+function addNumber() {
+  let condition = true;
+  while (condition) {
+    valueChoice = parseInt(prompt("Inserisci un numero intero "));
+
+    if (!isNaN(valueChoice) && Number.isInteger(valueChoice) && valueChoice !== 0) {
+      condition = false;
+    } else {
+      console.log("Per favore, inserisci solo numeri interi.");
+    }
+  }
+  return valueChoice;
+}
+
+function isPerfectNumber(num) {
+  let sum = 0;
+  for (let i = 1; i <= num / 2; i++) {
+    if (num % i === 0) {
+      sum += i;
+    }
+  }
+  return sum === num;
+}
+
+function perfectNumber(valueChoice) {
```

Comment:
Since this is a **print function** it might be useful to name it as `print....` or `display....`.